lighty-modules/lighty-gnmi/lighty-gnmi-test/src/test/java/io/lighty/modules/gnmi/test/gnmi/SimulatorCrudTest.java

```diff
@@ -51,29 +51,25 @@
     private static final int UPDATE_MTU_VAL = 500;
 
     private static SessionProvider sessionProvider;
-    private static SimulatedGnmiDevice target;
+    private static GnmiSimulatorApp target;
 
 
     @Before
     public void setUp() throws NoSuchAlgorithmException, CertificateException, InvalidKeySpecException, IOException,
             URISyntaxException {
-
-        target = new SimulatedGnmiDeviceBuilder().setHost(TARGET_HOST).setPort(TARGET_PORT)
-                .setInitialConfigDataPath(INITIAL_DATA_PATH + "/config.json")
-                .setInitialStateDataPath(INITIAL_DATA_PATH + "/state.json")
-                .setYangsPath(TEST_SCHEMA_PATH)
-                .build();
-        target.start();
+        target = new GnmiSimulatorApp();
+        target.start(false,
+            new String[]{"-c","src/test/resources/json/example_config.json"});
```

Comment:
use simulator builder rather, this would add more complexity for tests, might causing issues running tests on different environments
-> revert